web-registry/src/components/templates/ProjectDetails/ProjectDetails.config.ts

```diff
@@ -0,0 +1,3 @@
+export const IMAGE_STORAGE_BASE_URL =
+  process.env.REACT_APP_IMAGE_STORAGE_BASE_URL;
+export const API_URI = process.env.REACT_APP_API_URI;
```

Comment:
It's actually a convention that we've recently started adopting in folder-driven components, to make large components more readable we're outputting the constants or configuration parameters to a Component.config.ts file.

I take the opportunity to apply it to other files of the ProjectDetails component.

That said, I think it would be good to centralize these global constants at the application level, so that the environment variables are read at a single point in the app, or other general configurations are defined! We'll talk about it soon